src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorFeatures.kt

```diff
@@ -253,7 +254,23 @@ object GardenVisitorFeatures {
             }
         }
         if (hasIngredients) {
-            list.add(" §7(§aCraftable!§7)")
+            val leftToCraft = amount - amountInSacks
+            list.add(" §7(")
+            list.add(
+                Renderable.optionalLink(
+                    "§aCraftable!",
+                    {
+                        if (Minecraft.getMinecraft().currentScreen is GuiEditSign) {
+                            LorenzUtils.setTextIntoSign("$leftToCraft")
+                        } else {
+                            //  Possibly extract to a RecipeAPI object?
+                            //  TBD whether internal name or 'display name' as itemName better suited here?
+                            HypixelCommands.recipe(internalName.itemName)
```

Comment:
It's better to do this so it shows the crafting recipe right away, rather than having to click on the item first.
```suggestion
                            HypixelCommands.viewRecipe(internalName.toString())
```

Reply:
Aye, thanks!